sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs

```diff
@@ -32,6 +32,7 @@ namespace SonarAnalyzer.UnitTest.TestFramework
     internal struct SolutionBuilder
     {
         private const string GeneratedAssemblyName = "project";
+        private const string InitSnippet = @"namespace System.Runtime.CompilerServices { public class IsExternalInit { } } // See https://github.com/dotnet/roslyn/issues/45510";
```

Comment:
maybe take out the comment and put it above?

Reply:
I was thinking to have it here in case we get an error (3 years from now) for duplicate type (since it will appear on the same line with the error).